keras_cv/models/backbones/backbone.py

```diff
@@ -145,71 +145,17 @@ def from_preset(calling_cls, *args, **kwargs):
             )(cls.from_preset.__func__)
 
     @property
-    def backbone_level_outputs(self):
-        """Backbone outputs at each resolution level for transfer learning."""
-        return None
-
-    @backbone_level_outputs.setter
-    def backbone_level_outputs(self, value):
-        self._backbone_level_outputs = value
-
-    def get_feature_extractor(self, min_level=None, max_level=None):
-        """Convert the application model into a model backbone for other tasks.
-
-        The backbone model will usually take same inputs as the original
-        application model, but produce multiple outputs, one for each feature
-        level. Those outputs can be feed to network downstream, like FPN and RPN.
-
-        The output of the backbone model will be a dict with int as key and
-        tensor as value. The int key represent the level of the feature output.
-        A typical feature pyramid has five levels corresponding to scales P3,
-        P4, P5, P6, P7 in the backbone. Scale Pn represents a feature map 2n
-        times smaller in width and height than the input image.
-
-        Args:
-            min_level: optional int, the lowest level of feature to be included
-                in the output. Default to model's lowest feature level (based on
-                the model structure).
-            max_level: optional int, the highest level of feature to be included
-                in the output. Default to model's highest feature level (based
-                on the model structure).
-
-        Returns:
-            a `tf.keras.Model` which has dict as outputs.
-        Raises:
-            ValueError: When the model is lack of information for feature level,
-            and can't be converted to backbone model, or the min_level/max_level
-            param is out of range based on the model structure.
+    def pyramid_level_inputs(self):
+        """Intermediate model outputs for transfer learning.
```

Comment:
I think it would be more precise to say "for feature extraction" than "for transfer learning," as this isn't necessarily tied to transfer learning (even though it often is in practice)